rasa/nlu/test.py

```diff
@@ -398,6 +398,33 @@ def evaluate_response_selections(
     }
 
 
+def add_confused_intents_to_report(
+    report: Dict, target_intents: Iterable[Any], predicted_intents: Iterable[Any]
+) -> Dict:
+
+    from sklearn.metrics import confusion_matrix
+    from sklearn.utils.multiclass import unique_labels
+
+    cnf_matrix = confusion_matrix(target_intents, predicted_intents)
+
+    indices = np.argsort(cnf_matrix, axis=1)
+    labels = unique_labels(target_intents, predicted_intents)
+    n_candidates = min(3, len(labels))
+
+    for label in labels:
+        if report.get(label):
```

Comment:
can this case actually happen? A label which is not part of the report?

Reply:
yes! you can predict `None` :)

Reply:
mhm, should we add `None` to the report then?

Reply:
since we report false positives, it should show up in the 'confused with', so will include that in the test, thanks!